.pylintrc

```diff
@@ -0,0 +1,614 @@
+       non-parent-init-called,
+       nonexistent-operator,
+       nonlocal-and-global,
+       nonlocal-without-binding,
+       nonstandard-exception,
+       nonzero-method,
+       not-a-mapping,
+       not-an-iterable,
+       not-async-context-manager,
+       not-callable,
+       not-context-manager,
+       not-in-loop,
+       notimplemented-raised,
+       oct-method,
+       old-division,
+       old-ne-operator,
+       old-octal-literal,
+       old-raise-syntax,
+       parameter-unpacking,
+       pointless-string-statement,
+       print-statement,
+       property-on-old-class,
+       raising-bad-type,
+       raising-non-exception,
+       raising-string,
+       range-builtin-not-iterating,
+       raw_input-builtin,
+       redefine-in-handler,
+       reduce-builtin,
+       redundant-keyword-arg,
+       redundant-unittest-assert,
+       reimported,
+       relative-import,
+       reload-builtin,
+       repeated-keyword,
+       return-arg-in-generator,
+       return-in-init,
+       return-outside-function,
+       round-builtin,
+       setslice-method,
+       signature-differs,
+       slots-on-old-class,
+       standarderror-builtin,
+       star-needs-assignment-target,
+       super-on-old-class,
+       too-few-format-args,
+       too-few-public-methods,
+       too-many-format-args,
+       too-many-function-args,
+       too-many-function-attributes,
+       too-many-locals,
+       too-many-star-expressions,
+       truncated-format-string,
+       undefined-all-variable,
+       undefined-loop-variable,
+       unexpected-keyword-arg,
+       unexpected-special-method-signature,
+       unichr-builtin,
+       unicode-builtin,
+       unnecessary-lambda,
+       unnecessary-pass,
+       unnecessary-semicolon,
+       unpacking-in-except,
+       unpacking-non-sequence,
+       unreachable,
+       unsubscriptable-object,
+       unsupported-binary-operation,
+       unsupported-membership-test,
+       unused-format-string-argument,
+       unused-format-string-key,
+       unused-wildcard-import,
+       used-before-assignment,
+       useless-else-on-loop,
+       using-cmp-argument,
+       using-constant-test,
+       wildcard-import,
+       xrange-builtin,
+       yield-inside-async-function,
+       yield-outside-function,
+       zip-builtin-not-iterating,
+       unused-argument,
+       unused-import,
+
+# Things we'd like to try.
+# Procedure:
+# 1. Enable a bunch.
+# 2. See if there's spurious ones; if so disable.
+# 3. Record above.
+# 4. Remove from this list.
+       # arguments-differ,
+       # attribute-defined-outside-init,
+       # fixme,
+       # import-error,
+       # invalid-name,
+       # missing-docstring,
+       # no-member,
+       # pointless-statement,
+       # protected-access,
+       # redefined-builtin,
+       # redefined-outer-name,
+       # super-init-not-called,
+       # too-many-arguments,
+       # unused-variable,
+       # wrong-import-order,
```

Comment:
All of those currently raise an error message for me. I don't think it makes sense to activate all the tests though. Things like `invalid-name` or `wrong-import-order` are not very useful and produce to much false positives.